app/models/concerns/turbo/broadcastable.rb

```diff
@@ -67,15 +67,69 @@
 # to render a turbo stream template with multiple actions.
 #
+# == Page refreshes
+#
+# You can broadcast a "page refresh" to all clients subscribed to a stream. This is useful when you need to
+# broadcast a lot of partial updates, and the entire content is not easily updatable using Turbo stream actions like replace, append, prepend, or remove.
+#
+# It's recommended to use page refreshes as the primary option for broadcasting changes.
+# This approach allows you to remove the coupling between models and views to render complex partial updates without sacrificing the desired high UI fidelity.
+# We hope this makes using the other available Turbo Stream actions less necessary.
+#
+# The +broadcast_refreshes+ class method configures the model to broadcast a "page refresh" on creates, updates, and destroys to a stream
+# name derived at runtime by the <tt>stream</tt> symbol invocation. Examples
+#
+#   class Board < ApplicationRecord
+#     broadcast_refreshes
+#   end
+#
+# In this example, when a board is created, updated, or destroyed, a Turbo Stream for a
+# page refresh will be broadcasted to all clients subscribed to the "boards" stream.
+#
+# This works great in hierarchical structures, where the child record touches parent records automatically to invalidate the cache:
+#
+#   class Column < ApplicationRecord
+#     belongs_to :board, touch: true # +Board+ will trigger a page refresh on column changes
+#   end
+#
+# You can also specify the streamable declaratively by passing a symbol to the +broadcast_refreshes_to+ method:
+#
+#   class Column < ApplicationRecord
+#     belongs_to :board
+#     broadcast_refreshes_to :board
+#   end
+#
+# To achieve more granular control, , you can also broadcast a "page refresh" to a stream name derived from the passed <tt>streamables</tt> by using
+# the instance-level methods <tt>broadcast_refresh_to</tt> or <tt>broadcast_refresh_later_to</tt>.
+# These methods are particularly useful when you want to trigger a page refresh for more specific scenarios. Example:
+#
+#   class Clearance < ApplicationRecord
+#     belongs_to :petitioner, class_name: "Contact"
+#     belongs_to :examiner,   class_name: "User"
+#
+#     after_create_commit :broadcast_refresh_later
+#
+#     private
+#       def broadcast_refresh_later
+#         broadcast_refresh_later_to examiner.identity, :clearances
+#       end
+#   end
+#
+# In this example, a "page refresh" is broadcast to the stream named "identity:2:clearances" after a new clearance is created.
+# All clients subscribed to this stream will refresh the page to reflect the changes.
+#
+# When broadcasting page refreshes, Turbo will automatically debounce multiple calls in a row to only broadcast the last one. This is meant for scenarios where you process records in mass. Because of the nature of such signals, it makes no sense to broadcast them repeatedly and individually.
+#
```

Comment:
Some minor tweaks:

```suggestion
#
# The +broadcast_refreshes+ class method configures the model to broadcast a "page refresh" on creates, 
# updates, and destroys to a stream name derived at runtime by the <tt>stream</tt> symbol invocation. Examples
#
#   class Board < ApplicationRecord
#     broadcast_refreshes
#   end
#
# In this example, when a board is created, updated, or destroyed, a Turbo Stream for a
# page refresh will be broadcasted to all clients subscribed to the "boards" stream.
#
# This works great in hierarchical structures, where the child record touches parent records automatically
# to invalidate the cache:
#
#   class Column < ApplicationRecord
#     belongs_to :board, touch: true # +Board+ will trigger a page refresh on column changes
#   end
#
# You can also specify the streamable declaratively by passing a symbol to the +broadcast_refreshes_to+ method:
#
#   class Column < ApplicationRecord
#     belongs_to :board
#     broadcast_refreshes_to :board
#   end
#
# For more granular control, you can also broadcast a "page refresh" to a stream name derived 
# from the passed <tt>streamables</tt> by using the instance-level methods <tt>broadcast_refresh_to</tt> or
# <tt>broadcast_refresh_later_to</tt>. These methods are particularly useful when you want to trigger 
# a page refresh for more specific scenarios. Example:
#
#   class Clearance < ApplicationRecord
#     belongs_to :petitioner, class_name: "Contact"
#     belongs_to :examiner,   class_name: "User"
#
#     after_create_commit :broadcast_refresh_later
#
#     private
#       def broadcast_refresh_later
#         broadcast_refresh_later_to examiner.identity, :clearances
#       end
#   end
#
# In this example, a "page refresh" is broadcast to the stream named "identity:<identity-id>:clearances" 
# after a new clearance is created. All clients subscribed to this stream will refresh the page to reflect
# the changes.
#
# When broadcasting page refreshes, Turbo will automatically debounce multiple calls in a row to only broadcast the last one. 
# This is meant for scenarios where you process records in mass. Because of the nature of such signals, it makes no sense to
# broadcast them repeatedly and individually.
```